platform/src/components/aws/remix.ts

```diff
@@ -461,12 +461,15 @@ export class Remix extends Component implements Link.Linkable {
           const vite = await import("vite");
           const viteConfig = await vite.loadConfigFromFile(
             { command: "build", mode: "production" },
-            file,
+            path.join(sitePath, file),
```

Comment:
There's a 3rd argument to `loadConfigFromFile` called `loadConfigFromFile` (https://vitejs.dev/guide/api-javascript.html#loadconfigfromfile) which I guess could be used here? So,

```ts
vite.loadConfigFromFile({ command: "build", mode: "production" }, file, sitePath)
```

Maybe this also sets `viteConfig.root` to the correct value at the same time.

Reply:
Nevermind, just tried this and the `configRoot` parameter doesn't do anything as far as I can tell